src/components/CustomDropdownComp.tsx

```diff
@@ -43,54 +47,82 @@ export default function CustomDropdownComp(props: CustomDropdownComponentProps):
     const [value, setValue] = useState<Option>();
 
     const createOption = (label: string, secondaryLabel: string, id: string, urlstring: string) => ({
-        label: <Label
-            DisplayName={label}
-            UrlString={urlstring}
-            ClassNamePrefix={props.classNamePrefix}
-            EnableAvatar={props.useAvatar}
-            SecondLabel={secondaryLabel}
-        />,
+        label: (
+            <Label
+                DisplayName={label}
+                UrlString={urlstring}
+                ClassNamePrefix={props.classNamePrefix}
+                EnableAvatar={props.useAvatar}
+                SecondLabel={secondaryLabel}
+            />
+        ),
         value: label,
+        // eslint-disable-next-line object-shorthand
```

Comment:
Why? Just leave id. What it wants is this
```
{
   value: label,
   id,
   secondLabel,
   url: urlstring
}
```

Reply:
Ah yes I'm not very familiar with that notation shorthand, so it confused me a bit. Will update it